frontend/model/chatroom/getters.js

```diff
@@ -171,7 +185,7 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }, roo
     const myIdentityId = rootState.loggedIn.identityContractID
     for (const contractID in chatRoomsInDetail) {
       const chatRoom = rootState[contractID]
-      if (chatRoom && chatRoom.attributes && chatRoom.members[myIdentityId]) {
+      if (getters.isJoinedChatRoom(contractID, myIdentityId)) {
```

Comment:
The logic has slightly changed here. Before it was checking for the existence of `chatRoom`, and if it didn't exist it would take the `else` branch. Now, if it doesn't exist, it will take the `if` branch. This completely changes the behavior.

Reply:
Yes, I'll double check. `getters.isJoinedChatRoom(contractID, myIdentityId)` should AFAICT be equivalent to `chatRoom && chatRoom.members && chatRoom.members[myIdentityId]`. However, the `.attributes` check might need to be added back if it's needed.